modular_splurt/code/modules/research/techweb/nodes/syndicate_nodes.dm

```diff
@@ -0,0 +1,2 @@
+/datum/techweb_node/syndicate_basic
+	design_ids = list("decloner", "borg_syndicate_module", "suppressor", "largecrossbow", "donksofttoyvendor", "donksoft_refill", "syndiesleeper", "ci-xray", "safemag")
```

Comment:
I feel like it might be better not to modularize this, or to perhaps modify the list(adding onto it rather than overwriting it) in an overriden New()

Reply:
I-... Have managed to completely forget that `LAZYADD` existed, my bad. Will correct soon